pkg/Library/NodeGraph/LayoutUpdater.js

```diff
@@ -9,8 +9,12 @@
 
 ({
 
+shouldUpdate({graph}) {
+  return graph;
+},
+
 update({graph, previewLayout, nodegraphLayout}) {
-  if (graph && graph.$meta.id === previewLayout?.id && graph.$meta.id === nodegraphLayout?.id) {
+  //if (graph.$meta.id === previewLayout?.id && graph.$meta.id === nodegraphLayout?.id) {
```

Comment:
i think this check was needed, because when switching to a different graph, layout and graph happen to not be corresponding...

Reply:
Thanks!